docs/BR.md

```diff
@@ -1595,10 +1589,22 @@ The issuing CA SHALL document in its Certificate Policy or Certification Practic
 
 ## 7.3 OCSP profile
 
+1. All OCSP responses MUST conform with RFC6960 and be the profile specified in RFC5019.
+2. Responses MUST be pre-produced as specified in RFC5019.
```

Comment:
@vanbroup I don't believe there's anything wrong with backdating the revocation. And since this is the Baseline Requirements, it only applies to TLS services and TLS hierarchy. Obviously, if we have an OCSP responder providing status for both TLS and non-TLS certificates, we need a minimal level of assurance for those TLS certificates. CAs that wish to provide less than that assurance (e.g. for non-TLS cases) likely can and should use a separate intermediate.

I'm not sure I understand your remark about your request spike. 3.5 days is already the maximum that Microsoft permits a CA to issue. You may wish to review those requirements again and check with Microsoft, since you should not see any additional requests increasing.